frontend/public/components/utils/link.tsx

```diff
@@ -33,6 +33,17 @@ export const namespacedPrefixes = [
   '/status',
 ];
 
+export const CLUSTER_SCOPED_ROUTES = [
+  '/*/ns/:ns',
+  '/*/all-namespaces',
+  '/overview',
+  '/dashboards',
+  '/settings',
+  '/api-explorer',
+  '/catalog',
```

Comment:
`/catalog` route is in dev perspective

Reply:
also do we need list all cluster-scope URL here? it looks like some URLs are missing here, for example if I go to following URLs , I will still stay in ACM perspective 
```
/k8s/cluster/core~v1~Namespace
/k8s/cluster/apiextensions.k8s.io~v1~CustomResourceDefinition
/k8s/cluster/config.openshift.io~v1~Infrastructure/cluster 
```
<img width="1349" alt="Screenshot 2023-01-05 at 3 11 57 PM" src="https://user-images.githubusercontent.com/12692381/210722711-2bf6b51e-c225-430e-9653-55f6f76453af.png">